src/external/CMakeLists.txt

```diff
@@ -105,6 +113,7 @@ target_include_directories(vita-toolchain INTERFACE "${CMAKE_CURRENT_SOURCE_DIR}
 option(YAML_CPP_BUILD_TESTS "Enable testing" OFF)
 option(YAML_CPP_BUILD_TOOLS "Enable parse tools" OFF)
 option(YAML_CPP_BUILD_CONTRIB "Enable contrib stuff in library" OFF)
+option(YAML_CPP_INSTALL "Enable generation of install target" ON)
```

Comment:
Why `ON`? We don't need installation.
The only thing that this PR should have is this line with `OFF` and the yaml-cpp submodule update.

Reply:
No, it's a static library, we do not need installing.

Reply:
oki i understund